### PR TITLE
Add new docs to table of contents

### PR DESCRIPTION
**Pull request by eric-lidong**
For recently added functions like model version deletion, ensembler deletion etc, they are missing from the current table of contents in gitbook. This PR aims to add them, also some docs that seems to be previously missing in the main doc, e.g. eidt routers, custom model and treatment service.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -30,11 +30,14 @@
   * [Serve Model Endpoint](module/model/user-guide/model\_version\_endpoint.md)
   * [Model Deployment and Serving](module/model/user-guide/model\_deployment\_serving.md)
   * [Batch Prediction](module/model/user-guide/batch\_prediction.md)
+  * [Model Deletion](module/model/user-guide/model_deletion.md)
```
Comment (krithika369) [inline SUMMARY.md:33]: Model deletion is currently hidden from the users as it is very slow (only version deletion is possible). Shall we leave it out of these docs?

**Review by krithika369 — APPROVED** [review]
Thanks for updating these docs, @eric-lidong ! Left one comment. LGTM.

**Comment by deadlycoconuts** [comment]
Whoaa thanks a lot Eric for this PR! Was about to get to it but you beat me to it ><"

File: SUMMARY.md (hunk 1 above)
Comment (eric-lidong) [reply SUMMARY.md]: Thanks for the catch! Removed it for now.

**Pull request merged**